src/Util.java

```diff
@@ -0,0 +1,98 @@
+
+import java.util.Arrays;
+import java.util.LinkedList;
+import java.util.List;
+
+public class Util {
+
+	/**
+	 * Create utility method, which counts even numbers in given collection;
+	 */
+	public int countEvenNumbers(List<? extends Number> list){
+		int count = 0;
+		for (Number number: list) {
+			if(number.doubleValue() % 2 == 0)
+				count++;
+		}
+		return count;
+	}
+
+	/**
+	 * Create utility method, which takes two lists and index and replaces element at given position
+	 * in the first collection with corresponding element from the second one,
+	 * without possibility to do vice versa;
+	 */
+	public <T> void replaceElement(List<T> list1, List<T> list2, int index){
+		list1.set(index, list2.get(index));
```

Comment:
Almost. 
Try to write here `list2.set(index, list1.get(index));`. 
It mustn't even compile.

Reply:
ok) it is The Get and Put Principle, isn't it?